src/pharmpy/modeling/estimation.py

```diff
@@ -126,8 +126,16 @@ def calculate_parameters_from_ucp(
     --------
     calculate_ucp_scale : Calculate the scale for conversion from ucps
     """
+    fixed_parameters_dict = {
+        p.name: p.init for p in model.parameters if p.fix and p.name not in ucps
+    }
```

Comment:
Could `model.parameter.fixed.inits` be used to get all fixed values?

Reply:
Yes that could be used instead. Changed this as well as added some tests for coverage